modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/ConfigFile.scala

```diff
@@ -88,9 +87,12 @@ object ConfigFile {
    * Command line parameters have the highest priority.
    * Then the provided file.
    * Then the application.conf file.
+   * Then the application-default.conf file.
    */
-  private def configFileFallbacks(in: TSConfig): TSConfig =
-    ConfigFactory.load(in.withFallback(ConfigFactory.load()))
+  private def configFileFallbacks(in: TSConfig): TSConfig = {
+    val defaultConf = ConfigFactory.load("application-default.conf")
```

Comment:
```suggestion
    val defaultConf = ConfigFactory.load("application.conf")
```
To be consistent with the rest of the codebase (although it should better be names `reference.conf`)